modules/Timestamps.lua

```diff
@@ -118,6 +118,8 @@ Prat:AddModuleToLoad(function()
 
   
 --@localization(locale="zhCN", format="lua_table", handle-subnamespaces="none", same-key-is-true=true, namespace="Timestamps")@
+      ["Other Formatting Options"] = "其他设置",
+		  ["Timestamp Text Format"] = "时间戳格式",
```

Comment:
These won't end up in the table, they need to be entered into the translation tool